### ISSUE TITLE
BUG: Career Page Revamp

### ISSUE THREAD

**Issue by ira-mane** [Nactore-Org/Nacto-Care#104]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What went down?

The website's career page needs a revamp. The button size is too large, taking up half of the page, and the user interface is neither user-friendly.
![image](https://github.com/Nactore-Org/Nacto-Care/assets/108334485/e807734e-853e-48b6-abb9-df99a9e0aa84)
Could You please assign me the issue ? I would like to work on it.

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

**Comment by Harshil-Jani** [comment]
@ira-mane Yes, I totally do agree on this. It was just supposed to be a quick roll out. But I have some plans for the career page. Are you aware of how the KYC is done on most of the financial apps ? We want to have a similar procedure for onboarding the nurses.

Idea is that when the nurse opens up the career page, She gets a long form where she puts in many details like Name, Email, Phone Number , City, Aadhar, PAN, Professional Transcript and then we want to integrate the aadhar verification and pan verification from backend and have a status check if it was approved or not.  For the Professional Transcript they can upload a document which will be approved manually from the legal team and also have a status check next to it.

So for the context of this Issue you can skip thinking about backend part but just help us create a form like component which will collect all the details and for last three field have a status check with appropriate colour

Accepted - Green
Pending - Yellow
Rejected - Red

But until we build proper backend, Also keep the Apply via Email option on top.

**Comment by ira-mane** [comment]
Okay , @Harshil-Jani  
1. I think I have understood the issue .So we need a form page where we can take nurse detail ,from that page nurse can apply for job. Okay I can work on the issue.
2.  Also I was thinking if I can divide this issue in two part ? One would be designing UI/UX and other frontend coding. Let me know about this so that I can create one another issue. and you can assign me both.
3. If Yes ,Can you assign me this issue under gssoc24 and also add level ?
Thankyou.

**Comment by Harshil-Jani** [comment]
@ira-mane Sure. That would be fine for me to split in two. Please create a figma file and push it in a new design directory once you are done with designs so that as a PR you can get scored for design.

**Comment by ansh-h-mehta** [comment]
Hey @Harshil-Jani  As per our discussion on #235, I am ready to take up this issue as you've mentioned that the existing assignee has stalled it. Also, please reply on my question mentioned on #235, thank you 😇.

**Comment by ira-mane** [comment]
Hello @ansh-h-mehta I haven't stalled working on the issue. I agree there has been a delay due to some uncertain circumstances.
But I will complete my PR by tomorrow.

**Comment by ansh-h-mehta** [comment]
Hi @ira-mane In #235 I had received invitation from @Harshil-Jani to pick this up, as he thought that you've stalled this issue. I am sorry for the confusion.
